### PR TITLE
style: Remove any extra spaces from md files

### PR DESCRIPTION
**Pull request by zbw-zbw**
## Summary
Remove any extra Spaces from md files.

## How did you test this change?
Preview md files.

### CODE REVIEW

**Comment by zbw-zbw** [comment]
@sebmarkbage pls review the pr, thanks.

**Comment by zbw-zbw** [comment]
@acdlite pls review the pr, thanks.

File: fixtures/packaging/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Manual Testing Fixtures
 
-This folder exists for **React contributors** only.  
+This folder exists for **React contributors** only.
```
Comment (mattiasnordqvist) [inline fixtures/packaging/README.md:3]: This changes how it's displayed

File: fixtures/packaging/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 If you use React, you don't need to worry about it.
 
-These fixtures verify that the built React distributions are usable in different environments.  
+These fixtures verify that the built React distributions are usable in different environments.
```
Comment (mattiasnordqvist) [inline fixtures/packaging/README.md:6]: This removes newline

**Comment by rickhanlonii** [comment]
Thanks for the PR, but we generally don't take PRs that only change formatting because they tend to introduce subtle changes that are easy to miss (as the comment above caught).

**Pull request closed without merging**